rust/Cargo.toml.in

```diff
@@ -53,6 +53,7 @@ md-5 = "~0.9.1"
 regex = "~1.4.2"
 lazy_static = "~1.4.0"
 base64 = "~0.13.0"
+rustls = { version="~0.20.3", features=["quic"] }
```

Comment:
what does the features=quic do here?

Reply:
It enables the quic thing of rustls cf https://docs.rs/rustls/latest/rustls/quic/index.html
So as to "decrypt" the first packets cf use of `rustls::quic::Keys`

Does that answer your question ?

Reply:
yeah, I think it would be good to add comment here

Reply:
comment added